charts/platform-tenant/values.yaml

```diff
@@ -150,7 +150,7 @@ components:
     # enable all istio CR's
     enabled: true
     # namespace istio is installed in
-    namespace: "istio-system"
```

Comment:
this might be a regression break in other charts

Reply:
This is what `platform-system` uses now.